lib/ui/text.dart

```diff
@@ -312,7 +1431,7 @@ class FontFeature {
   int get hashCode => hashValues(feature, value);
 
   @override
-  String toString() => 'FontFeature($feature, $value)';
+  String toString() => 'FontFeature(\'$feature\', $value)';
```

Comment:
nit: consider either double quoting the outer or inner string rather than escaping.

Reply:
changed outer quotes to `"`.

I'm pretty mixed on this. I kinda like the consistency of always using `'`, and don't mind the `\` so much. I find it confusing to have mixed quoting styles and it gets particularly annoying when dealing with copy/paste. But I also like avoiding escapes to some extent, it does look a bit cleaner. I dunno.